lambda/app/src/routes.ts

```diff
@@ -122,10 +114,7 @@ export const setRoutes = (app: any) => {
 
   app.post(`${BASE_PATH}/requests-all`, async (req, res) => {
     try {
-      const session = await authenticate(req.headers);
-      if (!session) return res.status(401).json({ success: false, message: 'not authorized' });
```

Comment:
it will always include `session` object in `req`, otherwise, it will be caught in line `90` and stop propagating.
and as this is the expected behaviour of Express.js, I needed to mock `lambda-api-router` to do so.

Reply:
Oh, I understand now, that is a useful refactor.